components/more_direct_channels/more_direct_channels.jsx

```diff
@@ -175,9 +186,13 @@ export default class MoreDirectChannels extends React.Component {
     handlePageChange(page, prevPage) {
         if (page > prevPage) {
             if (this.listType === 'any') {
-                this.props.actions.getProfiles(page + 1, USERS_PER_PAGE);
+                this.props.actions.getProfiles(page + 1, USERS_PER_PAGE).then(() => {
+                    this.toggleChannelLoadingState(false);
+                });
             } else {
-                this.props.actions.getProfilesInTeam(page + 1, USERS_PER_PAGE);
+                this.props.actions.getProfilesInTeam(page + 1, USERS_PER_PAGE).then(() => {
```

Comment:
Does the `getProfilesInTeam` first argument should be `teamId` ([ref](https://github.com/mattermost/mattermost-redux/blob/master/src/actions/users.js#L366))?

Reply:
Yes, It should be.